tonic/src/transport/service/reconnect.rs

```diff
@@ -13,6 +14,7 @@ use tracing::trace;
 pub(crate) struct Reconnect<M, Target>
 where
     M: Service<Target>,
+    M::Error: Debug,
```

Comment:
I think we actually want to use `Into<Box<dyn Error + Send + Sync>>` here and then convert it. If it impls that trait and you call `into()` then it will automatically impl `Debug` because of the `Error` trait.

Reply:
I think I've done what you've suggested in the last comment.